src/pages/gd/Claim/ClaimBalance.tsx

```diff
@@ -40,13 +40,17 @@ export const ClaimBalance = () => {
                 .catch(() => undefined),
         [chainId]
     )
-    const formattedTime = useMemo(() => format(claimTime, 'hh aaa'), [claimTime])
+    const formattedTime = useMemo(() => claimTime && format(claimTime, 'hh aaa'), [claimTime])
     const { switchNetwork } = useEthers()
 
-    const network = useMemo(() => SupportedChains[chainId], [chainId])
+    const network = useMemo(
+        () => (chainId === (SupportedChains.FUSE as number) ? SupportedChains[42220] : SupportedChains[122]),
```

Comment:
maybe just 
```suggestion
        () => SupportedChains [chainId],
```

and do not think memo is required for so simple logic